packages/core/src/test/testRunner.ts

```diff
@@ -84,7 +84,8 @@ export async function runTests(
     })
 
     const dist = path.resolve(root, 'dist')
-    const testFile = process.env['TEST_FILE']?.replace('.ts', '.js')
+    const rawTestFile = process.env['TEST_FILE']
+    const testFile = rawTestFile?.replace(/\\/g, '/').replace('.ts', '.js')
```

Comment:
What is passing `\` paths here? We should not be doing that in any of our launch configs, build steps, etc. It's 100% unnecessary.

This change is probably still worth having, though.

Reply:
Ok, it's probably vscode itself passing the `\` paths.